lib/miq_pglogical.rb

```diff
@@ -1,90 +1,58 @@
 require 'pg'
-require 'pg/pglogical'
-require 'pg/pglogical/active_record_extension'
+require 'pg/logical_replication'
 
 class MiqPglogical
   include Vmdb::Logging
 
-  REPLICATION_SET_NAME = 'miq'.freeze
-  NODE_PREFIX = "region_".freeze
+  PUBLICATION_NAME = 'miq'.freeze
   ALWAYS_EXCLUDED_TABLES = %w(ar_internal_metadata schema_migrations repl_events repl_monitor repl_nodes).freeze
 
   attr_reader :configured_excludes
 
   def initialize
-    @connection = ApplicationRecord.connection
+    @connection = ApplicationRecord.connection.raw_connection
```

Comment:
As discussed, we may want to change this variable name to something like `raw_connection`, or `pg_connection`.  Most readers would expect a var named `connection` to be an ActiveRecord connection, but this isn't anymore.